player/command.c

```diff
@@ -507,7 +507,10 @@ static int mp_property_path(void *ctx, struct m_property *prop,
     MPContext *mpctx = ctx;
     if (!mpctx->filename)
         return M_PROPERTY_UNAVAILABLE;
-    return m_property_strdup_ro(action, arg, mpctx->filename);
+    char *path = mp_normalize_path(mpctx, mpctx->filename);
```

Comment:
Also I though you want to normalize `mpctx->filename` itself, not only the property.

Reply:
Nah, I decided that was too risky.